cmd/opentelemetry/app/exporter/elasticsearchexporter/dependencystore/dependency_store.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dependencystore
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"strings"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/jaegertracing/jaeger/cmd/opentelemetry/app/exporter/elasticsearchexporter/esclient"
+	"github.com/jaegertracing/jaeger/model"
+	"github.com/jaegertracing/jaeger/plugin/storage/es/dependencystore/dbmodel"
+	"github.com/jaegertracing/jaeger/storage/dependencystore"
+)
+
+const (
+	dependencyType          = "dependencies"
+	dependencyIndexBaseName = "jaeger-dependencies"
+
+	timestampField = "timestamp"
+
+	defaultDocCount = 10_000
```

Comment:
In the previous impl, it had a comment `// the default elasticsearch allowed limit`. Is that still the case? If so should probably have the comment here aswell.